website/docs/r/rde.html.markdown

```diff
@@ -185,6 +186,55 @@ should have `resolve=false` to avoid being resolved).
 In this last scenario, it is advisable to mark `resolve_on_removal=true` so Terraform can delete the RDE even if it was not
 resolved by anyone.
 
+<a id="metadata"></a>
+## Metadata
+
+-> Due to Terraform limitations, when performing a change in any of the metadata entries, a `terraform plan` shows that
+all of them will be deleted and re-created. However, the succeeding `terraform apply` will respect the non-changed metadata entries 
+by not modifying them.
+
+~> The provider configuration block `ignore_metadata_changes` is not compatible with metadata entries of `vcd_rde`.
+
+The `metadata_entry` is a set of metadata entries that have the following structure:
+
+* `key` - (Required) Key of this metadata entry.
+* `value` - (Required) Value of this metadata entry.
+* `type` - (Optional) Type of this metadata entry. One of: `StringEntry`, `NumberEntry`, `BoolEntry`. Defaults to `StringEntry`.
+* `namespace` - (Optional) Namespace of the metadata entry. Allows having multiple entries with same key in different namespaces.
+* `domain` - (Optional) Only meaningful for providers. Allows them to share entries with their tenants. Currently, accepted values are: `TENANT`, `PROVIDER`. Defaults to `TENANT`.
+* `readonly` - (Optional) `true` if the metadata entry is read only. Defaults to `false`.
+* `persistent` - (Optional) `true` if the metadata is persistent. Persistent entries can be copied over on some entity operation
+  (e.g. Creating a copy of a VDC, capturing a vApp to a template, instantiating a catalog item as a VM...). Defaults to `false`.
```

Comment:
This example, given that we are in the RDE documentation, is potentially confusing. Is there an example related to RDE that we can give?

Reply:
I need to check, as far as I know RDEs can't be re-used/inherited elsewhere.

Reply:
I see two possibilities:

1. We move this section to a location (such as a file in the `guides`) where the definition can apply to several resources. (BRW, if the API mention VMs, it is possible that we could start dealing with VM metadata using OpenAPI, but it is not the moment to try it).
2. We put a generic description, saying "not applicable to RDE as of v3.11".